Modules/Filtering/ImageFeature/include/itkHessian3DToVesselnessMeasureImageFilter.h

```diff
@@ -61,7 +61,6 @@ namespace itk
  * Yoshinobu Sato, Shin Nakajima, Hideki Atsumi, Thomas Koller,
  * Guido Gerig, Shigeyuki Yoshida, Ron Kikinis.
  *
- * http://www.spl.harvard.edu/archive/spl-pre2007/pages/papers/yoshi/
```

Comment:
You should replace it by this link:
https://link.springer.com/chapter/10.1007/BFb0029240

If you can find a link to full PDF, it would be even better.

Reply:
I believe http://www.image.med.osaka-u.ac.jp/member/yoshi/paper/linefilter.pdf will do the trick.